sqlstore/src/main/resources/db/migration/V8501__oxford_nanopore.sql

```diff
@@ -0,0 +1,102 @@
+DROP TABLE IF EXISTS ContainerQC;
+DROP TABLE IF EXISTS OxfordNanoporeContainer;
+DROP TABLE IF EXISTS FlowCellVersion;
+DROP TABLE IF EXISTS PoreVersion;
+DROP TABLE IF EXISTS RunOxfordNanopore;
+
+CREATE TABLE RunOxfordNanopore(
+  runId bigint(20) NOT NULL AUTO_INCREMENT,
+  minknowVersion varchar(100),
+  protocolVersion varchar(100),
```

Comment:
`minknowVersion` and `protocolVersion` don't seem to be attributes on the RunOxfordNanopore class; are they used/set elsewhere?